### PR TITLE
update windows-10-enterprise-subscription-activation.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #1448

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-10-enterprise-subscription-activation.md [hunk 1]
```diff
@@ -154,6 +154,8 @@ changepk.exe /ProductKey %ProductKey%
 )
 </pre>
 
+If no firmware-embedded Windows 10 activation key presents, the license will escalate to Windows 10 Enterprise using Subscription Activation directly.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-10-enterprise-subscription-activation.md]:
```suggestion
If a firmware-embedded Windows 10 activation key is not present, the license will escalate to Windows 10 Enterprise using Subscription Activation directly.
```

Comment (nenonix) [inline windows/deployment/windows-10-enterprise-subscription-activation.md]:
```suggestion
If no firmware-embedded Windows 10 activation key is present, the license will automatically step-up to Windows 10 Enterprise using Subscription Activation.
```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 , @nenonix ,
the sentence was updated, thank you very much!

**Comment by mypil** [comment]
@greg-lindsay - Can you please assist to check, approve and merge the content of this PR based on issue #1448?

Thank you.

**Pull request merged**